en/navigation_v2.ftl

```diff
@@ -0,0 +1,93 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+
+navigation-v2-mozilla = { -brand-name-mozilla }
+navigation-v2-download-firefox = Download { -brand-name-firefox }
+navigation-v2-get-a-firefox-account = Get a { -brand-name-firefox-account }
+navigation-v2-menu = Menu
+
+## Firefox menu
+
+navigation-v2-firefox-browsers = { -brand-name-firefox } Browsers
+navigation-v2-close-firefox-browsers-menu = Close { -brand-name-firefox } Browsers menu
+navigation-v2-firefox-for-desktop = { -brand-name-firefox } for Desktop
+navigation-v2-get-the-non-profit-backed = Get the non-profit-backed browser on { -brand-name-windows }, { -brand-name-mac-short } or { -brand-name-linux }.
```

Comment:
Should we change all `non-profit` to `not-for-profit`? I believe this is the current expression we use.